packages/vscode-extension/src/utilities/android.ts

```diff
@@ -2,21 +2,33 @@ import path from "path";
 import os from "os";
 import fs from "fs";
 
-export const ANDROID_HOME =
-  process.env.ANDROID_HOME || path.join(os.homedir(), "Library/Android/sdk");
 
-const ANDROID_STUDIO_PATH = "/Applications/Android Studio.app";
+export const ANDROID_HOME = process.env.ANDROID_HOME || (process.platform === "win32")
+  ? path.join(os.homedir(), "AppData/Local/Android/Sdk")
+  : path.join(os.homedir(), "Library/Android/sdk");
+
+const ANDROID_STUDIO_PATH = (process.platform === "win32")
+  ? "C:\\Program Files\\Android\\Android Studio"
+  : "/Applications/Android Studio.app";
 
 function findJavaHome() {
   // we try to use java bundled with Android Studio if exists
   // in newer distributions java is placed under Contents/jbr/Contents/Home and with some
   // older (but still recent ones) it is under Contents/jre/Contents/Home
   // we check if the first path exists and if not we try the second one
-  const jbrPath = path.join(ANDROID_STUDIO_PATH, "Contents/jbr/Contents/Home");
+
+  const jbrPath = (process.platform === "win32")
```

Comment:
might be nice to create a similar utill `getPlatform() ` as you did for webview.

Reply:
or maybe isWindows() and isMacOS()

Reply:
I've implemented `Platform` class that takes care of selecting the right paths depending on the platform.